Kickstarter-iOS/Views/Controllers/ProjectNavigatorViewController.swift

```diff
@@ -166,14 +167,28 @@ extension ProjectNavigatorViewController: ProjectPamphletViewControllerDelegate
                                   velocity: recognizer.velocity(in: scrollView),
                                   isDragging: scrollView.isTracking)
   }
+
+  func projectPamphletViewController(_ projectPamphletViewController: ProjectPamphletViewController,
+                                     didTapBackThisProject project: Project,
+                                     refTag: RefTag?) {
+    let rewardsViewController = RewardsCollectionViewController.instantiate(with: project, refTag: refTag)
+
+    let navigationController = UINavigationController(rootViewController: rewardsViewController)
+
+    self.present(navigationController, animated: true)
+  }
 }
 
+// MARK: - UIGestureRecognizerDelegate
+
 extension ProjectNavigatorViewController: UIGestureRecognizerDelegate {
   func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
     return true
```

Comment:
Is this for the hidden scrollview? Do we want to return true only for that scrollview's gesture?

Reply:
This is not new, I think it's for the pull-down effect in the project screen.